media-gfx/geeqie/geeqie-2.6.ebuild

```diff
@@ -85,6 +85,7 @@ src_configure() {
 		$(meson_feature zip archive)
 	)
 
+	filter-lto
```

Comment:
Please add a comment referencing the bug above it.

Reply:
done